src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -517,29 +519,51 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Return New LazyInvalidExpression(children, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
-        Private Function CreateBoundTryCastOperation(boundTryCast As BoundTryCast) As IConversionExpression
-            Dim operand As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundTryCast.Operand))
+        Private Function CreateBoundTryCastOperation(boundTryCast As BoundTryCast) As IOperation
             Dim syntax As SyntaxNode = boundTryCast.Syntax
+            Dim type As ITypeSymbol = boundTryCast.Type
+            Dim isImplicit As Boolean = boundTryCast.WasCompilerGenerated
+
+            Dim constantValue As [Optional](Of Object) = ConvertToOptional(boundTryCast.ConstantValueOpt)
+            Dim conversionInformation = CreateConversionOperand(boundTryCast.Operand, boundTryCast.ConversionKind, boundTryCast.Syntax, boundTryCast.Type)
+            Dim methodSymbol As MethodSymbol = conversionInformation.methodSymbol
```

Comment:
`methodSymbol` is not used in this method or the next method.

Reply:
Fixed.

---
In reply to: [143330388](https://github.com/dotnet/roslyn/pull/22056#discussion_r143330388) [](ancestors = 143330388)